### PR TITLE
Max tolerance error feature

### PR DESCRIPTION
**Pull request by poulNA**
To display the maximum error value depending on the given tolerance to be reported for all fields individually. 

**Questions for the review**:

@MTCam 
Please add a PR description to check off this box.
- [x] Is the scope and purpose of the PR clear?
  - [x] The PR should have a description.
~~- [ ] The PR should have a guide if needed (e.g., an ordering).~~
- [x] Is every top-level method and class documented? Are things that should be documented actually so?
- [x] Is the interface understandable? (I.e. can someone figure out what stuff does?) Is it well-defined?
- [x] Does the implementation do what the docstring claims?
~~- [ ] Is everything that is implemented covered by tests?~~
~~- [ ] Do you see any immediate risks or performance disadvantages with the design? Example: what do interface normals attach to?~~

### CODE REVIEW

**Comment by matthiasdiener** [comment]
Closes #508.

File: mirgecom/simutil.py [hunk 1]
```diff
@@ -505,11 +507,17 @@ def compare_files_vtu(
...
-                                 "values with given tolerance")
+            test_err = abs(arr1.GetValue(j) - arr2.GetValue(j))
+            if test_err > max_field_errors[i]:
+                max_field_errors[i] = test_err
+        print(f"Max Error: {max_field_errors[i]}")
+
+    violation = any([max_field_errors[i] > tolerance for i in range(nfields)])
+    if violation:
+        raise ValueError("Fidelity test failed: Mismatched data array "
+                                 "values with given tolerance. ")
```
Comment (MTCam) [inline mirgecom/simutil.py:521]: Would be good to include the tolerance value here.  Could be just:
f"Mismatched data array values {tolerance=}."

Comment (poulNA) [reply mirgecom/simutil.py:521]: Fixed! And the field names were Matthias' idea :)

**Review by MTCam — CHANGES_REQUESTED** [review]
Looks good to me!   Please add a PR description and then ready-to-merge.  I'm not just being nit-picky here, I get yelled at by the boss if I make reviews that don't satisfy all the check boxes :)

**Comment by MTCam** [comment]
Thanks so much Paul!

**Review by MTCam — APPROVED** [review]
👍

**Pull request merged**